config/libvirt-setup/roles/libvirt-system/templates/cloud-config/user-data

```diff
@@ -0,0 +1,25 @@
+#cloud-config
+#password: fedora
+#chpasswd: {expire: False}
+#ssh_pwauth: True
+ssh_authorized_keys:
+  - "ssh-rsa {{ item.value.admin_ssh_rsa }}"
+
+# Add users to the system. Users are added after groups are added.
+users:
+  - name: admin
+    gecos: Admin User
+    groups: wheel
+    lock-passwd: false
+    passwd: {{ item.value.admin_passwd }}
+    ssh-authorized-keys:
+      - ssh-rsa {{ item.value.admin_ssh_rsa }}
+
+timezone: US/Eastern
```

Comment:
(forgot to make this part of my review) can we use the timezone that ci.centos.org uses? Just so the console log timestamps in jenkins line up with test output from the libvirt host